cypress/support/commands.js

```diff
@@ -23,3 +23,12 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (host) => {
```

Comment:
I think it will be helpful if we can get #499 merged and use the `login` command added there. Cypress recommends having one login flow test using the actual UI, but then for subsequent tests to make whatever network requests/app state changes are necessary to have someone logged in so subsequent tests can be much faster